oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/util.js

```diff
@@ -56,3 +61,48 @@ export const eliminateUsedLanguages = (
   );
   return remainingLanguages;
 };
+
+const removeKeyFromNestedObjects = (inputObject) => {
+  const processArray = (arr) => {
+    return arr.map((item) => {
+      if (_isObject(item)) {
+        return _omit(item, "__key");
+      }
+      return item;
+    });
+  };
+
+  const processObject = (obj) => {
```

Comment:
What happens when there is a `__key` key in the top-level `inputObject`? It doesn't seem to me that it gets removed here. It removes `__key` only from child objects present in array-typed values only. Additionaly, cannot the `removeNullAndUnderscoreProperties` also take care of this stuff? :) It is just an another kind of underscore prop